build/shared/verify.Native.MacOS.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+set -e
+
+config=$1
+arch=$2
+folder=$3
+
+SCRIPT_PATH="$( cd "$(dirname "$0")" ; pwd -P )"
+. $SCRIPT_PATH/../$config-$arch/settings.sh
+
+
+verifyNative() {
+    local name=$1
+
+    local file=${folder}/Release${name}/$arch/Magick.Native-${name}-$arch.dll.$EXTENSION
+
+    if [ ! -f $file ]; then
+        echo "Unable to find $file"
+        exit 1
+    fi
+
+    ld_arch=$arch
+    if [ $ld_arch == "x64" ]; then
+        ld_arch="x86_64"
+    fi
+    exit_code=0
+    output=$(lipo -info $file 2>&1) || exit_code=$?
+    if [ $exit_code -ne 0 ]; then
+        echo "Failed to execute lipo: $output"
+        exit 1
+    fi
+
+    if echo "$output" | grep "is architecture: $ld_arch"; then
+        echo "Verified architecture $file"
+    else
+        echo "Unexpected architecture for $file"
+        echo $output
+        exit 1
+    fi
+
+    macos_version=$(sw_vers -productVersion)
+    # When specifying a platform version to ld, an error occurs, and so we need to specify the path to the
+    # system shared libraries to apply -lSystem
+    # This assumes that Xcode is installed, the path for Commandline tools is different
```

Comment:
Not sure if we need this part of the comment since we are running in GitHub actions?

Reply:
Sure, I can remove it. Do you mean just the "This assumes that Xcode is installed" comment, or the whole thing?

Reply:
I just meant that part the rest adds valuable information.

Reply:
removed and force pushed